app/ui.R

```diff
@@ -98,9 +98,9 @@ sidebar <- tags$div(
     ),
     checkboxGroupInput(
       "aheads",
-      "Forecast Horizon (Weeks)",
-      choices = AHEAD_OPTIONS,
-      selected = AHEAD_OPTIONS[1],
+      "Forecast Horizon (Days)",
+      choices = HOSPITALIZATIONS_AHEAD_OPTIONS,
+      selected = HOSPITALIZATIONS_AHEAD_OPTIONS[1L],
```

Comment:
checking my understanding: this sidebar panel is displayed for both normal and archive evaluations tabsets (because `condition = "input.tabset == 'evaluations' | input.tabset == 'evaluations_archive'"` [up on L59](https://github.com/cmu-delphi/forecast-eval/blob/71e1fb42e937a13c84eaa101d54964ed64ded59f/app/ui.R#L59)), but the startup configuration is for normal mode, yeah? Which is why we're using hospn aheads here instead of global aheads.

Reply:
Yep, the "current" tab is [specified for startup here](https://github.com/cmu-delphi/forecast-eval/blob/71e1fb42e937a13c84eaa101d54964ed64ded59f/app/ui.R#L156) and it contains only hosp data at the moment, so we need to make sure the sidebar startup values match.